src/App.jsx

```diff
@@ -1,37 +1,48 @@
 import React, { useEffect } from 'react';
-import { Route, Routes } from 'react-router-dom';
+import { Route, Routes, useNavigate } from 'react-router-dom';
 import { useDispatch, useSelector } from 'react-redux';
 import SignUp from './components/SignUp';
 import Home from './components/Home';
 import Index from './components/Index';
 import Reservations from './components/Reservations/Reservations';
 import Reserve from './components/Reserve';
 import Login from './components/Login';
+import AddHotel from './components/AddHotel';
 import { login } from './redux/login/login';
-import { getAllRoomsAsync } from './redux/room/room';
+import { getAllHotelsAsync } from './redux/hotel/hotel';
+import { getAllCitiesAsync } from './redux/city/city';
 import LogedUsers from './components/accessibility/LogedUsers';
 import DetailsView from './components/Details/DetailsView';
 
 const tokenKey = 'token';
 
 const App = () => {
   const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const loggedIn = useSelector((state) => state.login.isLoggedIn);
 
   useEffect(() => {
     if (localStorage.getItem(tokenKey)) {
       dispatch(login(JSON.parse(localStorage.getItem(tokenKey))));
     }
 
-    dispatch(getAllRoomsAsync());
+    dispatch(getAllHotelsAsync());
+    dispatch(getAllCitiesAsync());
```

Comment:
- [ ] Fetching data should be done only when the component that consumes it is mounted. In the case of `getAllHotelsAsync` you should get them on `Index` component gets mounted